package.json

```diff
@@ -8,14 +8,15 @@
     "start": "node index.js"
   },
   "engines": {
-    "node": "6.5.0",
-    "npm": "3.10.7"
+    "node": "^6.5.0",
+    "npm": "^3.10.7"
```

Comment:
Prefer to have explicit engine requirements rather than variables.  Not a big deal.